src/components/product/ProductCard/ProductCard.tsx

```diff
@@ -59,10 +59,10 @@ function ProductCard({
 
   return (
     <UICard
-      className="product-card"
-      data-card-variant={variant}
-      data-card-bordered={bordered}
+      data-fs-product-card-variant={variant}
+      data-fs-product-card-bordered={bordered}
       data-card-out-of-stock={outOfStock}
+      data-fs-product-card
```

Comment:
not so important, but what do you think about placing the data-attr that represents the component as the first?

Reply:
Sure thing, let's do this.